contact.html

```diff
@@ -176,10 +176,9 @@ <h3>Contact us</h3>
               </div>
               <div class="full white_fonts">
                 <p>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
-                  laboris nisi ut aliquip
+                 Education is the process of facilitating learning, or the acquisition of knowledge, skills, values, beliefs, and habits.
+                             Educational methods include teaching, training, storytelling, discussion and directed research

```

Comment:
Check formatting

Reply:
It is a good practice to be more verbose with the commit subject. Reviewers should be able to understand what code will be about without reading the PR description by just reading the commit subject.

Reply:
Yeah definitely!! I will take care of it next time!